rotkehlchen/chain/substrate/manager.py

```diff
@@ -505,6 +497,17 @@ def _set_chain_properties(self, node_interface: SubstrateInterface) -> None:
         self.chain_properties = self._get_chain_properties(node_interface)
         return None
 
+    def attempt_connections(self) -> None:
```

Comment:
Now this this method is "public", as it is called for kicking off the connections, for instance when adding the first KSM address.